theano/sparse/tests/test_basic.py

```diff
@@ -484,7 +484,7 @@ def test_adv_sub1_sparse_grad(self):
 
         # Test the sparse grad
         valm = numpy.random.rand(5, 4).astype(config.floatX)
-        valv = numpy.random.random_integers(0, 4, 10)
+        valv = numpy.random.randint(0, 4, 10)
```

Comment:
I just checked with an older version of numpy, and the same numbers are indeed generated with `random_integers(0, 4)` and `randint(0, 5)`, if the seed is the same.